AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/Model/KeyStoreAdmin.smithy

```diff
@@ -256,13 +259,23 @@ structure VersionKeyInput {
 structure VersionKeyOutput {
 }
 
-@documentation("
-Starts a Mutation to all Items of a Branch Key ID.
-Versions the Branch Key ID, such that the new version only has existed in the final state.
+@documentation(
+"Starts a Mutation to all Items of a Branch Key ID.
 Mutates the Beacon Key.
+Either Mutates the Active & its version (decrypt only), or versions the Branch Key,
+depending on the 'Do Not Version' argument.
+Regardless, if operation is successful,
+the Beacon, Active, & the Active's version are in the terminal state.
 Establishes the Mutation Commitment; Simultaneous conflicting Mutations are prevented by the Mutation Commitment.
 Mutations MUST be completed via subsequent invocations of the Apply Mutation Operation,
-first invoked with the Mutation Token returned in InitializeMutationOutput.")
+first invoked with the Mutation Token returned in 'Initialize Mutation Output'.
+This operation is idempotent-ish;
```

Comment:
let's remove the "ish". Either we say it is or not mention it. I don't think we want our javadocs to say something that reads halfbaked

Reply:
Half baked is not the goal.
Accurate is the goal.

But I have another idea:
```
With respect to the output's Mutation token, this operation is idempotent;
if invoked with the same request as an in-flight Mutation,
the operation will return successful
with the same Mutation Token as earlier requests.
```